cli/src/commands/version.rs

```diff
@@ -0,0 +1,13 @@
+use clap::Parser as ClapParser;
+
+/// Prints version information
+///
+/// This subcommand is implemented for backward compatibility reasons.
+/// Its behavior should be the same as that of the `--version` flag which
+/// is handled internally by clap.
+#[derive(Debug, ClapParser)]
+pub struct Opts {}
+
+pub fn run(_opts: Opts) {
+    println!("subxt-cli {}", clap::crate_version!());
```

Comment:
Thanks for having a go at this, the PR looks good! 
I'm wondering if this information would suffice to identify subxt versions for debugging purposes. 
Substrate provides a similar [versioning](https://github.com/paritytech/substrate/blob/e9b0facf70eeb08032cc7e83548c62f0b4a24bb1/utils/build-script-utils/src/version.rs#L52) including the commit hash as well.

Reply:
done